cibyl/config.py

```diff
@@ -96,8 +142,6 @@ def from_path(path):
         :type path: str or None
         :return: The configuration instance.
         :rtype: :class:`Config`
-        :raise ConfigurationNotFound: If no definition could be
-            retrieved.
```

Comment:
'from_file' still throws this error, which means there is a chance that this function does too.

Reply:
Add also the new 'EmptyConfiguration' error.